pages/common/svn-changelist.md

```diff
@@ -0,0 +1,24 @@
+# svn changelist
+
+> Associate a changelist with a set of files.
+> More information: <http://svnbook.red-bean.com/en/1.7/svn.advanced.changelists.html>.
+
+- Add files to a [new] changelist:
```

Comment:
Why not?
```suggestion
- Create a changelist:
```

Reply:
Or:
```suggestion
- Create a changelist with specific files:
```

Reply:
> Why not?

I want to express that either a new changelist is created or files are added to an existing one